<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Umi-OCR</string>
  </property>

  <!-- 主窗口中心部件 -->
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>

    <!-- 标签页容器 -->
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>

      <!-- 截图 OCR 页面 -->
      <widget class="QWidget" name="pageScreenshotOcr">
       <attribute name="title">
        <string>截图OCR</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutScreenshot">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>

      <!-- 批量图片 OCR 页面 -->
      <widget class="QWidget" name="pageBatchOcr">
       <attribute name="title">
        <string>批量图片</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutBatchOcr">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>

      <!-- 批量文档 OCR 页面 -->
      <widget class="QWidget" name="pageBatchDoc">
       <attribute name="title">
        <string>批量文档</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutBatchDoc">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>

      <!-- 二维码页面 -->
      <widget class="QWidget" name="pageQrcode">
       <attribute name="title">
        <string>二维码</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutQrcode">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>

      <!-- 任务管理器页面 -->
      <widget class="QWidget" name="pageTaskManager">
       <attribute name="title">
        <string>任务管理</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_TaskManager">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_TM_Header">
          <item>
           <widget class="QLabel" name="label_title">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>任务管理器</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_TM">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pause_all">
            <property name="text">
             <string>暂停全部</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_resume_all">
            <property name="text">
             <string>恢复全部</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_clear_completed">
            <property name="text">
             <string>清空已完成</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>782</width>
             <height>542</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_cards">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <spacer name="verticalSpacer_TM">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>

      <!-- 设置页面 -->
      <widget class="QWidget" name="pageSettings">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_Settings">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="listWidget_sidebar">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget_pages">
          <widget class="QWidget" name="page_general">
           <layout class="QVBoxLayout" name="verticalLayout_general">
            <item>
             <widget class="QLabel" name="label_general">
              <property name="text">
               <string>常规设置</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_general">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_ocr_engine">
           <layout class="QVBoxLayout" name="verticalLayout_engine">
            <item>
             <widget class="QLabel" name="label_engine">
              <property name="text">
               <string>OCR 引擎设置</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_engine">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_cloud">
           <layout class="QVBoxLayout" name="verticalLayout_cloud">
            <item>
             <widget class="QLabel" name="label_cloud">
              <property name="text">
               <string>云服务设置</string>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_cloud">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>

     </widget>
    </item>

   </layout>
  </widget>

  <!-- 状态栏 -->
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>

 </widget>
</ui>
